### ISSUE TITLE
Outlook 2007: Calendar Should Provide Summary of X appointments on Specified Date

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2942]
**Reported by kevinchao89 on 2013-01-24 16:06**
Win7, and NVDA 2012.3.1

Open Outlook 2007;
CTRL+2 for Calendar;
Actual: NVDA reads current focused date/time
Expected: in addition to what is currently read, provide x appointments summary
RIGHT/LEFT ARROW to NEXT/PREV dates
Actual: NVDA reads current focused date/time
Expected: in addition to what is currently read, provide x appointments summary

**Comment by leonardder** [comment]
This sounds like a general request to give more information in Outlook calendar view.

**Comment by derekriemer** [comment]
can someone test this and see if outlook now does this? I think it might. cc @josephsl

**Comment by josephsl** [comment]
Hi, I need help from at least one another tester who uses Outlook 2010 and 2013, as well as steps to reproduce on newer systems. Thanks.

**Comment by PratikP1** [comment]
I can verify this in Outlook 2016 (version 16.0.8201.2102). I can also verify in Outlook 2013. Quickly tested on a colleague's laptop. All relevant updates are installed.

**Comment by josephsl** [comment]
Hi, also, last time I used it, Calendar UWP provides this info as well. Thanks.

**Comment by fernando-jose-silva** [comment]
Sometimes the nvda reads the appointments when walking with the arrows, the right and the left however for many times it fails and to see the appointments it is necessary to press tabe.

**Comment by Adriani90** [comment]
I think this could have negative effects on performance of NVDA while navigating through the calendar. NVDA is quite slugish when the calendar is quite full. Try to implement the holydays addin in ms outlook and you will see what I mean. I would rather recommend to change the view to active or list. You will then see all your appointments in a list. The appointments are sorted chronologically. In active view, the first apointment is the nex outstanding. In the list view, the first appointment is the very first which you have created or where you have been invited in ms outlook. it could be five years ago. You can sort your appointments by date, type and so on.

**Comment by fernando-jose-silva** [comment]
Respectfully I would like to disagree, other screen readers report the appointments with the right and left arrows on the calendar and there is no slowness.
I did tests in outlook 2016 and managed to get nvda to inform the appointments correctly, and with agility.
This makes my previous comment no longer valid.
I have not tested on other outlook and windows verifications.

**Comment by feerrenrut** [comment]
It seems like this issue can be closed.
